compiler/test/suites/types.re

```diff
@@ -43,9 +43,8 @@ describe("aliased types", ({test}) => {
     "type_alias_5",
     {|
       type Foo<a> = (String, List<a>)
-      let foo = (x: Foo<Number>) => {
+      let foo = (x: Foo<Number>) =>
         x: (String, List<Number>)
-      }
```

Comment:
Yes, that's why. If you wanted to keep the block but have that annotation, you could write it like this:
```
let foo = x => {
  (x: (String, List<Number>))
}
```
or like this:
```
let foo = x => {
  (x): (String, List<Number>)
}
```

If we ever decide to change our annotation syntax to use `::` (and I don't think anyone objected to it, it just needed to be implemented, so we could just do it now), then it wouldn't be a problem, and it would look like this:
```
let foo = x => {
  x :: (String, List<Number>)
}
```

Reply:
Can we keep the curlies using the 2nd example and add a TODO (issue?) about updating the test when we change to double colons (we should really change to double colons in 0.5...)